pkg/cortex/runtime_config.go

```diff
@@ -26,8 +26,6 @@ type runtimeConfigValues struct {
 
 	Multi kv.MultiRuntimeConfig `yaml:"multi_kv_config"`
 
-	IngesterChunkStreaming *bool `yaml:"ingester_stream_chunks_when_using_blocks"`
```

Comment:
Could you run `make doc` ?

Reply:
This is not documented anywhere :(

https://github.com/cortexproject/cortex/issues/5153

Reply:
Oh yeah. Just in this page: https://cortexmetrics.io/docs/configuration/v1guarantees/